e2e/scaffolding/extrinsicHelpers.ts

```diff
@@ -1,3 +1,4 @@
+import '@frequency-chain/api-augment';
```

Comment:
Not sure how this was working without this before, but it has to be called before polkadot/api